app/scripts/controllers/threebox.js

```diff
@@ -103,13 +110,23 @@ class ThreeBoxController {
     })
   }
 
+  async _checkThreeBoxBackupExistance () {
+
+  }
+
   async new3Box () {
     const accounts = await this.keyringController.getAccounts()
-    const address = accounts[0]
-
-    if (this.getThreeBoxSyncingState()) {
+    this.address = await this.keyringController.getAppKeyAddress(accounts[0], 'wallet://3box.metamask.io')
+    let backupExists
+    try {
+      const threeBoxConfig = await Box.getConfig(this.address)
+      backupExists = threeBoxConfig.spaces && threeBoxConfig.spaces.metamask
+    } catch (e) {
+      log.error(e)
+      backupExists = false
+    }
+    if (this.getThreeBoxSyncingState() || backupExists) {
```

Comment:
Is `this.getThreeBoxSyncingState()` redundant in this case? I'm not sure it's possible to get to this point unless it's truthy

Reply:
Not redundant, as `initializeThreeBox` is called on import